examples/java/psg-spring-boot-demo/src/main/java/io/psg/demo/service/DataService.java

```diff
@@ -0,0 +1,75 @@
+package io.psg.demo.service;
+
+import io.psg.demo.model.TransactionStatus;
+import iog.psg.service.common.CredentialsMessage;
+import iog.psg.service.metadata.*;
+import lombok.extern.java.Log;
+import net.devh.boot.grpc.client.inject.GrpcClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+@Service
+@Log
+public class DataService {
+    @Value("${clientId}")
+    private String clientId;
+    @Value("${token}")
+    private String token;
+
+    @GrpcClient("psg-services")
+    private MetadataServiceGrpc.MetadataServiceBlockingStub metadataService;
+
+    public List<TransactionStatus> getMetadata() {
+        ListMetadataRequest request = ListMetadataRequest.newBuilder()
+                .setCredentials(getCredentials())
```

Comment:
Let's enrich this example with timestamps to demonstrate full-specified request

Reply:
Fixed)